Makefile

```diff
@@ -83,25 +83,30 @@ gocilint:
 
 ## exec: Run given command. e.g; make exec run="go test ./..."
 exec:
-	GOBIN=$(GOBIN) $(run)
+	@$(GOENVVARS) $(run)
 
 ## clean: Clean build files. Runs `go clean` internally.
 clean:
-	@-rm $(GOBIN)/ 2> /dev/null
+	@-rm -r $(GOBIN) 2> /dev/null
 	@echo "  >  Cleaning build cache"
-	$(GOENVVARS) go clean
+	@$(GOENVVARS) go clean
 
 ## build: Build the project.
-build: install
+build: clean install
 	@echo "  >  Building Hermez binary..."
 	@bash -c "$(MAKE) migration-pack"
-	$(GOENVVARS) go build $(LDFLAGS) -o $(GOBIN)/$(GOBINARY) $(GOCMD)
+	@$(GOENVVARS) go build $(LDFLAGS) -o $(GOBIN)/$(GOBINARY) -x $(GOCMD)
```

Comment:
After run this Makefile tasks in a new machine the -x parameters makes it very verbose bringing in a lot of info. Could we remove it or make it optional?